platform/os_windows.go

```diff
@@ -72,40 +72,18 @@ func GetProcessSupport() error {
 	return err
 }
 
+var tickCount = syscall.NewLazyDLL("kernel32.dll").NewProc("GetTickCount64")
+
 // GetLastRebootTime returns the last time the system rebooted.
 func GetLastRebootTime() (time.Time, error) {
-	out, err := exec.Command("cmd", "/c", "wmic os get lastbootuptime").Output()
-	if err != nil {
-		log.Printf("Failed to query wmic os get lastbootuptime, err: %v", err)
-		return time.Time{}.UTC(), err
-	}
-
-	lastBootupTime := strings.Split(strings.TrimSpace(string(out)), "\n")
-	if strings.TrimSpace(lastBootupTime[0]) != "LastBootUpTime" || len(lastBootupTime) != 2 {
-		log.Printf("Failed to retrieve boot time")
-		return time.Time{}.UTC(), fmt.Errorf("Failed to retrieve boot time with 'wmic os get lastbootuptime'")
-	}
-	systemBootupTime := strings.Split(lastBootupTime[1], ".")[0]
-
-	// The systembootuptime is in the format YYYYMMDDHHMMSS
-	bootYear := systemBootupTime[0:4]
-	bootMonth := systemBootupTime[4:6]
-	bootDay := systemBootupTime[6:8]
-	bootHour := systemBootupTime[8:10]
-	bootMin := systemBootupTime[10:12]
-	bootSec := systemBootupTime[12:14]
-	systemBootTime := bootYear + "-" + bootMonth + "-" + bootDay + " " + bootHour + ":" + bootMin + ":" + bootSec
-
-	log.Printf("Formatted Boot time: %s", systemBootTime)
-
-	// Parse the boot time.
-	layout := "2006-01-02 15:04:05"
-	rebootTime, err := time.ParseInLocation(layout, systemBootTime, time.Local)
```

Comment:
Will this cause a difference in output in different regions?

Reply:
based on timezone, its printing the reboot time. 

pst:

2021/08/09 13:19:34 [12816] Formatted Boot time: 2021-07-20 15:02:32

i then set it to ist:

2021/08/10 01:49:58 [17720] Formatted Boot time: 2021-07-21 03:32:32